.github/workflows/code-quality.yml

```
@@ -0,0 +1,212 @@
+                body: comment
+              });
+            }
+      
+      - name: Check for critical issues
+        run: |
+          # Check if there are any TypeScript errors
+          if npm run type-check 2>&1 | grep -q "error"; then
+            echo "❌ TypeScript errors found"
+            exit 1
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)